### PR TITLE
Gitlab integration page updates

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by kbatuigas — APPROVED** [review]

File: jekyll/_cci2/gitlab-integration.adoc [hunk 1]
```diff
@@ -3,22 +3,36 @@ contentTags:
...
 :icons: font
 :experimental:
 
+This page walks you through integrating a GitLab project with CircleCI. The sections below introduce you to concepts and ways to manage CI/CD (continuous integration and continuous delivery) pipelines for your GitLab project. CircleCI features that are in development for GitLab projects are detailed in the <<coming-soon>> section.
+
 [#overview]
-== Overview
+== What is GitLab CI/CD?
+
+GitLab CI/CD allows you to build, test (continuous integration) and deploy (continuous deployment, continuous delivery) your projects each time a developer pushes a change to the codebase, and/or when a merge request is created. As an alternative, you can integrate with CircleCI to manage your CI/CD pipelines. If you use GitLab (SaaS or Self-managed) to and manage version control and store your code, GitLab CI/CD is enabled for your project by default.
```
Comment (rosieyohannan) [inline jekyll/_cci2/gitlab-integration.adoc:18]:
```suggestion
GitLab CI/CD allows you to build, test (continuous integration) and deploy (continuous deployment, continuous delivery) your projects each time a developer pushes a change to the codebase, and/or when a merge request is created. As an alternative, you can integrate with CircleCI to manage your CI/CD pipelines. If you use GitLab (SaaS or Self-managed) for version control and code storage, GitLab CI/CD is enabled for your project by default.
```

**Review by kbatuigas — APPROVED** [review]
🚀

File: jekyll/_cci2/gitlab-integration.adoc [hunk 2]
```diff
@@ -3,22 +3,36 @@ contentTags:
...
 
+This page walks you through integrating a GitLab project with CircleCI. The sections below introduce you to concepts and ways to manage CI/CD (continuous integration and continuous delivery) pipelines for your GitLab project. CircleCI features that are in development for GitLab projects are detailed in the <<coming-soon>> section.
+
 [#overview]
-== Overview
+== What is GitLab CI/CD?
+
+GitLab CI/CD allows you to build, test (continuous integration) and deploy (continuous deployment, continuous delivery) your projects each time a developer pushes a change to the codebase, and/or when a merge request is created. As an alternative, you can integrate with CircleCI to manage your CI/CD pipelines. If you use GitLab (SaaS or Self-managed) for version control and code storage, GitLab CI/CD is enabled for your project by default.
+
+Integrate with CircleCI to access features for automating software delivery:
```
Comment (rosieyohannan) [inline jekyll/_cci2/gitlab-integration.adoc:20]:
```suggestion
Integrate with CircleCI to access features for automated software delivery:
```

[the remark this answers is not in the document]
Comment (rosieyohannan) [reply jekyll/_cci2/gitlab-integration.adoc]: Done, and "security" is already there in the list of features

File: jekyll/_cci2/gitlab-integration.adoc [hunk 3]
```diff
@@ -790,3 +800,10 @@ The xref:status-badges#[status badge] and xref:insights-snapshot-badge#[Insights
 == Next Steps
 - xref:config-intro#[Configuration tutorial]
 - xref:hello-world#[Hello world]
+
+[#further-reading]
+== Further reading
```
Comment (rosieyohannan) [inline jekyll/_cci2/gitlab-integration.adoc:805]:
```suggestion
[#relevant-gitlab-articles]
== Relevant GitLab articles
```

**Pull request merged**